app/views/topics/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for @topic do |f| %>
```

Comment:
You defined ```topic``` so you should use it instead of ```@topic```. If you want to stick with ```@topic``` you can remove ```topic: @topic```